src/components/Quiz/QuizSummary.tsx

```diff
@@ -30,12 +25,21 @@ const QuizSummary: React.FC<IProps> = ({
   const valueStyles = { fontWeight: "700", mb: 2 }
   const labelStyles = { fontSize: "sm", m: 0, color: "disabled" }
 
-  // Render QuizSummary component
   return (
     <Box w="full" fontSize={["xl", "2xl"]}>
-      <Text fontWeight="700" textAlign="center">
-        {isPassingScore ? "You passed the quiz!" : "Your results"}
+      <Text
+        fontWeight="700"
+        textAlign="center"
+        color={isPassingScore ? "success500" : "default"}
```

Comment:
What is `default`? is it a color or should it be `transparent`?

`success500` is the old theme color.
```suggestion
        color={isPassingScore ? "success" : "default"}
```

Reply:
Fixed. `default` is the text color, now using `body` instead